log/slog/slog.go

```diff
@@ -72,22 +74,37 @@ func mapToSlice(m map[string]any) []any {
 
 // Info logs a line with level info using the loggerWrapper instance.
 func (l *loggerWrapper) Info(msg string, args ...any) {
+	if l.sensitive {
```

Comment:
Yes, this is intentional since I want some logging to be done even in case of sensitivity, because some logs is better for debugging than no logs.

The exercise of using the args (that appear as `key=value` in the logs) properly is left upto the end user.

Reply:
Another option would be, instead of removing the args, we can replace the values as `*`. What do you think?

Reply:
No Panache folks made it clear the values needs to be completely removed

Reply:
This is marked as resolved. Are we moving forward with this?